netkan/netkan/download_counter.py

```diff
@@ -133,7 +241,7 @@ def commit_counts(self) -> None:
             [self.output_file.as_posix()],
             'NetKAN Updating Download Counts'
         )
-        logging.info('Download counts changed and commited')
+        logging.info('Download counts changed and committed')
         self.ckm_repo.git_repo.remotes.origin.push('master')
```

Comment:
I can't believe I managed to persist misspelling 'committed' through 2 rewrites of this service :laughing: